tests/conftest.py

```diff
@@ -536,7 +536,8 @@ def factory(
             assert result.exception is not None, result.output
             assert result.exit_code != 0, result.exit_code
         else:
-            assert result.exception is None, result.output
+            import traceback
+            assert result.exception is None, ''.join(traceback.format_exception(*result.exc_info))
             assert result.exit_code == 0, result.exit_code
```

Comment:
Nice! I wonder if here we could also print stderr?

Reply:
We could, but if the code excepts, I think the stack trace is the most useful anyway

Reply:
Oh, sorry meant the assert below, for non-zero exit codes

Reply:
ah I see, yeah that makes sense, will add it